web/app/controllers/bloggs_controller.rb

```diff
@@ -12,17 +12,19 @@ def show
 
   def new
     @article = Blogg.new
+    authorize! :new, @article
   end
 
   def edit
     puts 'EDITING'
     puts params
     @article = Blogg.find(params[:id])
+    authorize! :edit, @article
```

Comment:
yeah i saw that before actually, i tried it briefly and couldn't get it to work as I wanted, so I just went the lazy route... you're right it should be fixed.  Also (to clarify terminology) while these are all methods, because they are within the resource controller & part of the "RESTful" methods set, they are called "actions"